web/queue.go

```diff
@@ -321,12 +321,15 @@ func (app *Application) serveQueueFailedJob(w http.ResponseWriter, req *http.Req
 	return nil
 }
 
-type jobqueueStats = *jobqueue.Stats
-type dispatcherStats = *dispatcher.Stats
+// JobQueueStats is an alias to pointer type of jobqueue.Stats.
+type JobqueueStats = *jobqueue.Stats
+
+// DispatcherStats is an alias to pointer type of dispatcher.Stats.
+type DispatcherStats = *dispatcher.Stats
 
 // Stats contains queue statistics and worker statistics.
 type Stats struct {
-	jobqueueStats
-	dispatcherStats
+	JobqueueStats
+	DispatcherStats
```

Comment:
> As a result of fixing a reflect bug, Unmarshal can no longer decode into fields inside embedded pointers to unexported struct types, because it cannot initialize the unexported embedded pointer to point at fresh storage. Unmarshal now returns an error in this case.
>
> https://tip.golang.org/doc/go1.10#encoding/json